phpstan.neon

```diff
@@ -27,6 +27,7 @@ parameters:
         - '#Call to an undefined method DOMNode::setAttribute\(\)#'
         - '#Binary operation "\+" between array|false and array results in an error#'
         - '#Result of method Cake\\Auth\\BaseAuthenticate::unauthenticated\(\) \(void\) is used#'
+        - '#Call to an undefined method DateTimeInterface::setTimezone\(\)#'
```

Comment:
Apparently the interface does not have the `setTimezone()` method but since both `DateTime` and `DateTimeImmutable` do and userland code can't implement the `DateTimeInterface` I don't see any practical problem in ignoring this.